docs/learn-parachains.md

```diff
@@ -79,6 +80,71 @@ Reserved balances are non-transferrable and cannot be used for staking. At the e
 DOT is unreserved. Parachains that have not secured a new lease to extend their slot will
 automatically become parathreads.
 
+## Common Good Parachains
+
+"Common Good" parachains are parachain slots reserved for functionality that benefits the the Web3
```

Comment:
Web3 is a bit broad here. The main point is that they help other layer 1 blockchains operate more effectively by reducing repeated logic that everyone would want to use. Having more efficient parachains benefits Web3, but that's kind of one step removed here.
```suggestion
"Common Good" parachains are parachain slots reserved for functionality that benefits the the Polkadot
```